### PR TITLE
Add profile_image_url column

### PR DESCRIPTION
**Pull request by waans11**
- Add one more column called profile_image_column to fetch a twitter user's profile image URL from the raw data

### CODE REVIEW

File: examples/twittermap/script/ingestMySqlTwitterToLocalCluster.sh [hunk 1]
```diff
@@ -52,6 +52,7 @@ $create_twitter_ds_tweet = "CREATE TABLE IF NOT EXISTS `twitter_ds_tweet` (
   `lang` varchar(255) NOT NULL,
   `user.lang` varchar(255) NOT NULL,
   `user.name` varchar(255) NOT NULL,
+  `user.profile_image_url` varchar(255) NOT NULL,
```
Comment (JavierJia) [inline examples/twittermap/script/ingestMySqlTwitterToLocalCluster.sh]: we should not touch the MySQL script. The data is generated differently.

File: examples/twittermap/script/ingestPostgreSqlTwitterToLocalCluster.sh [hunk 2]
```diff
@@ -35,6 +35,7 @@ create table if not exists "twitter_ds_tweet" (
              "is_retweet" smallint not null,
              "text" text not null,
              "user.screen_name" varchar(255) not null,
+             "user.profile_image_url" varchar(255) not null,
```
Comment (JavierJia) [inline examples/twittermap/script/ingestPostgreSqlTwitterToLocalCluster.sh]: ditto.

File: examples/twittermap/noah/src/main/resources/twitter/aql/ddl.aql [hunk 3]
```diff
@@ -6,6 +6,7 @@ create type typeUser if not exists as open {
     id: int64,
     name: string,
     screen_name : string,
+    profile_image_url : string,
```
Comment (JavierJia) [inline examples/twittermap/noah/src/main/resources/twitter/aql/ddl.aql]: and let's use this change to remove those misleading index creation as well:

```
//"using" "compaction" "policy" CompactionPolicy ( Configuration )? )?		 //"using" "compaction" "policy" CompactionPolicy ( Configuration )? )?
 create index text_idx if not exists on ds_tweet("text") type keyword;		 create index text_idx if not exists on ds_tweet("text") type keyword;
 create index location_idx if not exists on ds_tweet(coordinate) type rtree;		 create index location_idx if not exists on ds_tweet(coordinate) type rtree;
 create index time_idx if not exists on ds_tweet(create_at) type btree;		 create index time_idx if not exists on ds_tweet(create_at) type btree;
 create index state_idx if not exists on ds_tweet(geo_tag.stateID) type btree;		 create index state_idx if not exists on ds_tweet(geo_tag.stateID) type btree;
 create index county_idx if not exists on ds_tweet(geo_tag.countyID) type btree;		 create index county_idx if not exists on ds_tweet(geo_tag.countyID) type btree;
 create index city_idx if not exists on ds_tweet(geo_tag.cityID) type btree;
```

Comment (waans11) [reply examples/twittermap/noah/src/main/resources/twitter/aql/ddl.aql]: Done

Comment (JavierJia) [reply examples/twittermap/noah/src/main/resources/twitter/aql/ddl.aql]: sorry, can you change the `keyword` to `fulltext` as well?

Comment (waans11) [reply examples/twittermap/noah/src/main/resources/twitter/aql/ddl.aql]: Done

**Pull request merged**